home-data-migration/src/main/resources/config/liquibase.properties

```diff
@@ -0,0 +1,8 @@
+changeLogFile=db/changelog/master.xml
+driver=org.postgresql.Driver
+url=jdbc:postgresql://localhost:5433/postgres
+username=postgres
+password=password
+promptOnNonLocalDatabase=false
+parameter.admin.email=admin@example.com
```

Comment:
Let's simplify the task for now and get rid of the idea to provide admin creds. Use some defaults.

Reply:
Done.